src/packages/sdk/src/IntrospectionEngine.ts

```diff
@@ -495,11 +502,7 @@ Please put that file into a gist and post it in Slack.
       id: messageId++,
       jsonrpc: '2.0',
       method,
-      params: [
-        {
-          ...params,
-        },
-      ],
+      params: params ? [params] : undefined,
```

Comment:
So the params were cloned before (using the ... spread), I am not sure why exactly.
Because of the `params: any` this change looks scary to me 😅

Reply:
Maybe we keep it like before, should be like this I guess 
```suggestion
      params: params ? [{...params}] : [],
```

Actually, why was this change needed?

Reply:
It was needed because the debugPanic RPC command required the params to be undefined not an `[]`

Reply:
Ok I've changed it to clone params if they are present otherwise it's `undefined`